lib/msal-browser/src/event/EventHandler.ts

```diff
@@ -71,4 +81,31 @@ export class EventHandler {
         }
     }
 
+    /**
+     * Emit account added/removed events when cached accounts are changed in a different tab or frame
+     */
+    private handleAccountCacheChange(e: StorageEvent): void {
+        try {
+            const cacheValue = e.newValue || e.oldValue;
+            if (!cacheValue) {
+                return;
+            }
+            const parsedValue = JSON.parse(cacheValue);
+            if (typeof parsedValue !== "object" || !AccountEntity.isAccountEntity(parsedValue)) {
+                return;
+            }
+            const accountEntity = CacheManager.toObject<AccountEntity>(new AccountEntity(), parsedValue);
+            const accountInfo = accountEntity.getAccountInfo();
+            if (!e.oldValue && e.newValue) {
+                this.logger.info("Account was added to cache in a different window");
+                this.emitEvent(EventType.ACCOUNT_ADDED, undefined, accountInfo);
```

Comment:
We can also emit this event whenever accounts are added in the same browsing context, though I thought it might be redundant with the `LOGIN_SUCCESS` event. Open to thoughts

Reply:
The event would not be emitted for that use case with the current implementation. But if we think that would be useful we can add it, I've been going back and forth on it myself so I'm very open to arguments for or against

Reply:
it sounds fine to add it for every account addition, it allows for more granular control over actions related to account CRUD.

Reply:
Given loading external tokens is a testing-only use case at the moment, we could add this later if it proves necessary.

Reply:
Where did we land here?

Reply:
We'll add the event for same-context account additions incrementally, if needed. Until then people can achieve the same result by listening to the `LOGIN_SUCCESS` or `LOGOUT_SUCCESS` events.